src/main/java/org/openmrs/addonindex/legacy/LegacyController.java

```diff
@@ -75,13 +75,14 @@ public String checkUpdate(@PathVariable("moduleId") String moduleId) throws Exce
 	@RequestMapping(method = RequestMethod.GET, value = "findModules")
 	public LegacyFindModulesResponse findModules(
 			@RequestParam(value = "sSearch", required = false) String query,
+			@RequestParam(value = "tags", required = false) String tag,
```

Comment:
Makes sense. I thought I'd do that but then I was like maybe we shall support tags in the future in OpenMRS especially since there's a Gsoc project already based on it. Moreover, I thought that since the parameter is optional, it wouldn't matter. But since you've mentioned, I'll change it's value to null and pass it